src/System.Windows.Forms/src/System/Windows/Forms/Controls/DataGridView/DataGridViewCell.cs

```diff
@@ -207,14 +207,11 @@ public string ErrorText
     [AllowNull]
     private string ErrorTextInternal
     {
-        get => (string?)Properties.GetObject(s_propCellErrorText) ?? string.Empty;
+        get => Properties.GetValueOrDefault(s_propCellErrorText, string.Empty)!;
```

Comment:
It might be good to change GetValueOrDefault to have NotNullIfNotNullAttribute on the return if the default value is not null. Then we could avoid the forgiving operator.

That said, the method would have to throw if the value in the dictionary is null and the default value is not null. So maybe not.

For now:

```suggestion
        get => Properties.GetValueOrDefault<string?>(s_propCellErrorText) ?? string.Empty;
```